impl/src/main/java/org/opencord/olt/impl/OltFlowService.java

```diff
@@ -1455,7 +1469,7 @@ public void event(FlowRuleEvent event) {
             }
         }
 
-        private void updateCpStatus(FlowRuleEvent.Type type, ConnectPoint cp, FlowRule flowRule) {
+        public void updateCpStatus(FlowRuleEvent.Type type, ConnectPoint cp, FlowRule flowRule) {
```

Comment:
why public ? probably protected is enough.

Reply:
done